.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,40 @@
+name: "Run Tests"
+
+on:
+  pull_request:
+
+jobs:
+  run_tests:
+    name: "Run Tests on PHP ${{ matrix.php_version }}"
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.experimental }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
```

Comment:
i take it the `ubuntu-latest` vs `ubuntu-18.04` is just about package availability?

Reply:
https://github.com/shivammathur/setup-php#cloud-osplatform-support